### PR TITLE
numatop: init at 2.1

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

Useful utility.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/os-specific/linux/numatop/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ numactl ncurses ];
+  checkInputs = [ check ];
+
+  doCheck  = true;
+
+  meta = with stdenv.lib; {
+    description = "observation tool for runtime memory locality characterization and analysis of processes and threads running on a NUMA system";
```
Comment (markuskowa) [inline pkgs/os-specific/linux/numatop/default.nix]:
```suggestion
    description = "Tool for runtime memory locality characterization and analysis of processes and threads on a NUMA system";
```

**Review by markuskowa — COMMENTED** [review]
The tool requires root access. Is this intended?

File: pkgs/os-specific/linux/numatop/default.nix (hunk 1 above)
Comment (dtzWill) [reply pkgs/os-specific/linux/numatop/default.nix]: thank you, was blanking on my attempts to make more concise.

**Comment by dtzWill** [comment]
> The tool requires root access. Is this intended?

I think so? :).

**Review by markuskowa — APPROVED** [review]

**Comment by markuskowa** [comment]
The platform should be restricted to `x86_64-linux`. Since the tool is provided by intel it may not be intended to work on ARM.

**Pull request merged**